examples/move/coin/sources/regcoin.move

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//docs::#regulate
+module examples::regcoin {
+    use sui::coin::{Self, DenyCap};
+    use sui::deny_list::{DenyList};
+
+    public struct REGCOIN has drop {}
+
+    fun init(witness: REGCOIN, ctx: &mut TxContext) {
+        let (treasury, deny_cap, metadata) = coin::create_regulated_currency(witness, 6, b"REGCOIN", b"", b"", option::none(), ctx);
+        transfer::public_freeze_object(metadata);
```

Comment:
I am not convinced we should recommend freezing metadata.
Applies to `my_coin.move` too.

Reply:
If you're not, I'm definitely not.